Standards/scs-0214-v1-k8s-node-distribution.md

```diff
@@ -80,15 +80,27 @@ If the standard is used by a provider, the following decisions are binding and v
   can also be scaled vertically first before scaling horizontally.
 - Worker node distribution MUST be indicated to the user through some kind of labeling
   in order to enable (anti)-affinity for workloads over "failure zones".
+- To provide data about this node distribution and be able to test and verify compliance with this standard,
+  providers MUST label their K8s nodes with the labels "topology.kubernetes.io/region",
+  "topology.kubernetes.io/zone" and "topology.scs.openstack.org/host-id"
```

Comment:
[Here](https://github.com/SovereignCloudStack/issues/issues/540#issuecomment-1997330524), @martinmo proposed changing the label name to `topology.scs.community/host-id`

As I do not have a strong opinion here, I just changed this name in the related e2e test https://github.com/SovereignCloudStack/k8s-cluster-api-provider/commit/ffb722b42e99977d6cb42817993e11558d816012

So use the same name or raise your voice in the related issue if you do not agree with the latest proposal.

Reply:
I agree. I think so does @cah-hbaum, but I don't know for sure.

Reply:
Also agree, I only had this information later on. I'm gonna change it with the next commit.